app/assets/stylesheets/variables/_base.scss

```diff
@@ -28,6 +28,7 @@ $even-lighter-gray: #f6f6f6;
 $light-background-color: $super-light-gray;
 $white: #fff;
 $gray-text: #777;
+$grayer-text: #ccc;
```

Comment:
To me "grayer" sounds darker than "gray" -- can we call it `$lighter-gray-text`?